index.html

```diff
@@ -43,6 +43,263 @@ <h1 class="primary-text">
+                        
+                    <li class="html-lang"> <img class="select-0 lang-img"src="./images/icons/html5-icon.png" alt="html icon"> HTML</li>
+                        
+                    <li class="css-lang"><img class="select-0 lang-img"src="./images/icons/css3-icon.png" alt="css icon"> CSS</li>
+                        
+                </ul>
+                <!--frameworks-->
+                <ul class="skill-lang-selector2 no-style">
+                       
+                    <li class="skill-name">
+                       <a href="#" class="skill-link">Frameworks</a>
+                    </li>     
+                    <li class="skill-link">
+                        <a href="#"><img class="skill-icon" src="./images/icons/button-icon-rt.png" alt="skill selector button"></a> 
+                    </li>
+                </ul>
+                <hr class="skills-line1"> 
+            
+                <!-- skill 2-->
+                <ul class="skill-lang-selector2 no-style">
+                       
+                        <li class="skill-name">
+                           <a href="#" class="skill-link">Skill</a>
+                        </li>
+                        
+                        
+                        <li class="skill-link">
+                            <a href="#"><img class="skill-icon" src="./images/icons/button-icon-rt.png" alt="skill selector button"></a> 
+                        </li>
+                </ul>
+                <hr class="skills-line2"> 
+            </div>
+            </section>        
+        </section>      
+                <!--Contact me section begins here -->
+                <footer>
+                    <section class="contact-me-section">
+                        <h2 class="primary-text">
+                            Contact me                    
+                        </h2>
+                        <p class="intro-text contact-text">
+                            If you have an application you are interested 
+                            in developing, a feature that you need built or a 
+                            project that needs coding. I&#39;d love to help with it 
+                        </p>
+                    <!--contact form begins here-->
+                        <form id="contact-form" name="contact-form" action="https://formspree.io/f/xqkozzva">
+                            <ul class="contact-form-items">
+                                <li class="sender-name">
+                                   <input id="username" class="username" name="user_name" type="text" maxlength="30" placeholder="Darilyn" required />
+                                </li>
+                                <li class="sender-email">
+                                    <input id="email" class="email" name="user_email" type="email"myemail@gmail.comgit  required/>
```

Comment:
- [ ] Kindly add placeholder to this input field according to the project requirement. 👍